README.md

```diff
@@ -6,10 +6,28 @@ Soapbox provides managed web application hosting services, encapsulating best-pr
 
 ### Requirements
 
- - Go 1.8 or greater
- - Ruby 2.2 or greater
- - PostgreSQL 9.5 or greater
- - Terraform 0.9.11 or greater
+ - Go 1.8 or greater - see directions below
+ - Ruby 2.2 or greater - Install using [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/rvm/install)
+ - PostgreSQL 9.5 or greater - [Ubuntu](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-16-04) | [Mac](https://solidfoundationwebdev.com/blog/posts/how-to-install-postgresql-using-brew-on-osx)
+ - Terraform 0.9.11 or greater - Download and install Terraform from [here](https://www.terraform.io/downloads.html).
+
+### Installing Go
+
+###### Linux
+
+``` shell
+$ bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
```

Comment:
My preference is not to document or suggest a non-standard third-party installer or environment manager such as this. They aren't necessarily common or standard in the Go community. There are already other places online documenting how to install Go, including the official Go site, which I would prefer to start people with. I don't want to imply that you need to use such tools in order to use Soapbox -- the dependency is Go, not gvm.